src/qtiCommonRenderer/renderers/interactions/TextEntryInteraction.js

```diff
@@ -111,7 +111,7 @@ var render = function render(interaction) {
                 messageType = 'info';
             }
 
-            showTooltip($input, messageType, message);
+            if (count) showTooltip($input, messageType, message);
```

Comment:
Did you align this behavior change with UX/business?

Besides, if you do that, some previous code becomes obsolete.

Reply:
Please also take care of the code style. Any block of code should be put between braces. Single line expressions are not allowed.

Reply:
Previous code does not become obsolete because you need to show tooltip in case you have a max char of 5 for example, and the user only writes 2 chars.

This `if (count)` is only to prevent show tooltip when `$input.val().length` as talked previously with UX/Business but you can check with @KRAMAOAT if you have doubts

Reply:
It does become, though. There's no action performed on `!count` condition, and yet we check it only at the very end, and even do things like `else message = __('%d characters llowed', maxChars);`.